[BUG] Cannot find module 'open-im-sdk-wasm/lib/types/entity'

### OpenIM Server Version

latest

### Operating System and CPU Architecture

macOS (AMD)

### Deployment Method

Source Code Deployment

### Bug Description and Steps to Reproduce

Typescript 5.x 以后新增了一个 `"moduleResolution": "bundler"` ，由于没有把 `types/entity.d.ts` 与 `types/params.d.ts` 在 `index.d.ts` 文件中导出，就会导致 `[ERROR] TS2307: Cannot find module 'open-im-sdk-wasm/lib/types/entity' or its corresponding type declarations.` 异常。

建议修改:

```diff
export { getSDK } from './sdk';
export { CbEvents } from './constant';
export * from './types/enum';
+ export * from './types/entity';
+ export * from './types/params';
+ export * from './types/eventData';
```

### Screenshots Link

_No response_

--- a/src/index.ts
+++ b/src/index.ts
@@ -2,3 +2,6 @@ export { getSDK } from './sdk';
 
 export { CbEvents } from './constant';
 export * from './types/enum';
+export * from './types/entity';
+export * from './types/eventData';
+export * from './types/params';

